wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -305,26 +324,53 @@ public void stop(String workspaceId, Map<String, String> options)
       throw new ConflictException(
           format("Could not stop workspace '%s' because its state is '%s'", workspaceId, status));
     }
+    String stoppedBy =
+        firstNonNull(
+            sessionUserNameOr(workspace.getAttributes().get(WORKSPACE_STOPPED_BY)), "undefined");
```

Comment:
Is it possible to have here the same method as WorkspaceManager has? I mean `sessionUserNameOrUndefined`?

Reply:
This is a bit another use-case